docs/content/usage/packages/container.en-us.md

```diff
@@ -39,6 +39,12 @@ Images must follow this naming convention:
 
 `{registry}/{owner}/{image}`
 
+Name your image according to the convetion with the following command:
+
+```shell
+docker tag {image}:{tag} gitea.example.com/{owner}/{image}:{tag}
+```
+
```

Comment:
What about:

```suggestion
When building your docker image, using the naming convention above, this looks like:
\```
docker build -t {registry}/{owner}/{image}:{tag}
\```

where your registry is the domain of your gitea instance (e.g. gitea.example.com)
```

as it gives a full example of the build command showing an example to use the complete naming convention

Reply:
I like that 👍

Reply:
+1 . The key point is about the tag naming, but not the tag command.

Reply:
````suggestion
Name your image according to the convetion with the following command:

```shell
docker build -t {registry}/{owner}/{image}:{tag} .
```

````

Reply:
👍🏼